src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -37,7 +37,7 @@ public class CommandTestUtil {
         }};
 
     public static final String VALID_NAME_AMY = "Amy Bee";
-    public static final String VALID_NAME_BOB = "Bob Choo";
+    public static final String VALID_NAME_BOB = "Bart Choo";
     public static final String VALID_PHONE_AMY = "11111111";
     public static final String VALID_PHONE_BOB = "22222222";
```

Comment:
May I ask if BOB is still relevant? Otherwise we have to standardise the name to BART.